bukkit/rpk-permissions-bukkit/src/main/kotlin/com/rpkit/permissions/bukkit/command/charactergroup/CharacterGroupCommand.kt

```diff
@@ -29,12 +29,14 @@ class CharacterGroupCommand(private val plugin: RPKPermissionsBukkit) : CommandE
 
     private val characterGroupAddCommand = CharacterGroupAddCommand(plugin)
     private val characterGroupRemoveCommand = CharacterGroupRemoveCommand(plugin)
+    private val characterGroupListCommand = CharacterGroupRemoveCommand(plugin)
```

Comment:
This should be CharacterGroupListCommand
```suggestion
    private val characterGroupListCommand = CharacterGroupListCommand(plugin)
```